zuul.d/edpm_multinode.yaml

```diff
@@ -457,13 +399,13 @@
             networks:
               default: {}
               internal-api:
-                trunk_parent: default
+                trunk-parent: default
```

Comment:
(non-blocking) question: Was this a typo?

Reply:
I'm not sure if it was a typo or not but it was a mistake, we have `trunk-parent` being used by our net_map module[1] and `trunk_parent` being used by ci-bootstrap role[2]. I assume the confusion originated from there. 

[1] https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Fci-framework+trunk-parent+path%3A%2F%5Etests%5C%2Funit%5C%2Fmodule_utils%5C%2F%2F&type=code
[2] https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Fci-bootstrap%20trunk_parent&type=code

Reply:
`trunk_parent` is the "old" name of the field from the original PR in the mapper that added that functionality. When I refactored that PR in a follow up to add the best effort strategy I changed the fields that were using `_`s as word separator to `-`-